crates/core/c8y_api/src/smartrest/message_ids.rs

```diff
@@ -0,0 +1,107 @@
+//! Definitions of Smartrest MQTT message ids.
+//!
+//! - https://cumulocity.com/docs/smartrest/smartrest-one/#built-in-messages
+//! - https://cumulocity.com/docs/smartrest/mqtt-static-templates/
+
+// Message IDs
+
+pub const ERROR: usize = 41;
+pub const JWT_TOKEN: usize = 71;
+
+// Static templates
+
+// Publish templates
+
+// Inventory templates (1xx)
+
+pub const DEVICE_CREATION: usize = 100;
```

Comment:
I wonder if it would help some to have the smartrest code added as a prefix.

```suggestion
pub const DEVICE_CREATION_100: usize = 100;
```

Reply:
I personally don't see the value in having the message ids in the suffix (it would help with developer education, but that's about it)...but if we switch to a new id, then it would just mean we have to switch out the whole variable name...but I don't have a strong preference either way...

Reply:
I agree with @reubenmiller in that I think it renders the constant somewhat redundant. It's pretty easy to see what the value of the constant is.